base/uk.ac.stfc.isis.ibex.ui.dae/src/uk/ac/stfc/isis/ibex/ui/dae/spectraplots/PersistedLocalChannel.java

```diff
@@ -80,4 +84,28 @@ public void onValue(T value) {
     public String getPVAddress() {
     	return address;
     }
+    
+    /**Attempt to write initial value to local PV. If fails (due to an existing race condition), try again until it passes.
+     * @param initialValue - value to be written to PV
+     */
+    private void write_to_pv(T initialValue) {
+    	Runnable task = new Runnable() {
+    		public void run() {
```

Comment:
This tells checkstyle it's an overridden method and so it doesn't need javadoc

```suggestion
                @Override
    		public void run() {
```